api/lib/utils.js

```diff
@@ -9,6 +9,7 @@ function renameCounterpartyToIssuer(amount) {
     amount.issuer = amount.counterparty;
     delete amount.counterparty;
   }
+  return amount;
```

Comment:
Yes, command-query separation is good. This is just a convenience so that you don't have to put the call on a separate line. I'm not sure if the convenience is worth violating command-query separation, but ultimately we want to get rid of this kind of mutation regardless.